code/components/jomjol_wlan/read_wlanini.cpp

```diff
@@ -119,6 +120,19 @@ void LoadWlanFromFile(std::string fn, char *&_ssid, char *&_password, char *&_ho
             }
         }
 
+        if ((zerlegt.size() > 1) && (toUpper(zerlegt[0]) == "HTTP_USERNAME")){
+            http_username = zerlegt[1];
+            if ((http_username[0] == '"') && (http_username[http_username.length()-1] == '"')){
```

Comment:
What happens if `zerlegt[1]` is the empty string? Or it has just a single `"` character in it? In both cases, you would end up with invalid memory access. It's probably a good idea to add a check like this:
```suggestion
            if (http_username.length() >= 2 http_username.front() == '"' && http_username.back() == '"')
```